lib/ossy/cli/changelogs/generate.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'ossy/cli/commands/core'
+require 'ossy/import'
+require 'ossy/types'
+
+require 'yaml'
+require 'tilt'
+require 'dry/struct'
+
+module Ossy
+  module CLI
+    module Changelogs
+      class Generate < Commands::Core
+        class Context < OpenStruct
+          def self.new(data)
+            super(releases: data.map(&Release.method(:new)))
+          end
+        end
+
+        class Release < Dry::Struct
+          transform_keys(&:to_sym)
+
+          ChangeList = Types::Array.of(Types::String).optional
+
+          attribute :version, Types::Version
+          attribute :date, Types::Date
+          attribute? :summary, Types::String.optional
+          attribute? :fixed, ChangeList
```

Comment:
```suggestion
          attribute :fixed, ChangeList
```

Reply:
I want those keys to be omittable actually